### ISSUE TITLE
Build 5.159 refuses to accept previous characters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chummer5a/chummer5a#49]
```
What steps will reproduce the problem?
1. Select a character made in a previous build of Chummer (incomplete or 
complete)
2.
3.

What is the expected output? What do you see instead?
I expect to be able to read and edit files.

What version of the product are you using? On what operating system?
Build 5.159 on Windows 7

Please provide any additional information below.
Please help. I've got a session this weekend and I want to be able to use 
Chummer.

```

Original issue reported on code.google.com by `rty1...@gmail.com` on 12 Mar 2015 at 8:20

**Comment by Neocyberman** [comment]
I also have this issue, reloading the settings folder still produces an exception

**Comment by chummer5a** [comment]
Resolved in 5.159 (Proper).
